lib/rubocop/cop/corrector.rb

```diff
@@ -9,6 +9,8 @@ module Cop
     # The nodes modified by the corrections should be part of the
     # AST of the source_buffer.
     class Corrector < ::Parser::Source::TreeRewriter
+      NOOP_CONSUMER = ->(diagnostic) {} # noop
```

Comment:
Good. I have a PR I need to finish with a better interface, but this is good in the meantime 👍